src/modules/file/file.controller.ts

```diff
@@ -175,4 +175,17 @@ export class FileController {
       return f;
     });
   }
+
+  @Put('move/:fileUuid/to/:folderUuid')
```

Comment:
At the very end, we are updating a resource field, so following REST, if we are moving or not, it does not matter, we are doing a partial update of a resource, thus, we focus on what is happening with the resource (GET, POST, PUT, PATCH, DELETE and so on). This is a partial update, thus, a PATCH. Using /move on the path is adding unnecesary overhead from my opinion, and distracting developers to think in no-convention path naming instead of in RESTful terms